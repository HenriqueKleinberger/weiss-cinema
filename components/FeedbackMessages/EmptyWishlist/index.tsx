import { EMPTY_WISHLIST } from '../../../constants/messages';
import MovieImg from '../../../public/logo.svg';
import { Container } from '../styles';

const EmptyWishlist = () => {
  return (
    <Container>
      <MovieImg width={100} height={100} />
      <p>{EMPTY_WISHLIST}</p>
    </Container>
  );
};

export default EmptyWishlist;
