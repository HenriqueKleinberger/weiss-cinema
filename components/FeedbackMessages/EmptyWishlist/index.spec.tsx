import React from 'react';
import { shallow } from 'enzyme';
import EmptyWishlist from './index';

describe('EmptyWishlist', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<EmptyWishlist />);
    expect(wrapper).toMatchSnapshot();
  });
});
