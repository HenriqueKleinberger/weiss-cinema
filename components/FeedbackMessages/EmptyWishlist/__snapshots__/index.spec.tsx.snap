// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmptyWishlist should match snapshot 1`] = `
<styled.div>
  <logo.svg
    height={100}
    width={100}
  />
  <p>
    Your wishlist is empty, save new movies of your interest
  </p>
</styled.div>
`;
